File: c#/sum.cs
// A Hello World! program in C#.
using System;
namespace HelloWorld
{
    class Hello 
    {
        static void Main() 
        {
            int Number1, Number2, Sum;
            Console.Write("To print the sum of two numbers, enter the first number:");
            Number1 = int.Parse(Console.ReadLine());
            Console.Write("Enter the second number:");
            Number2 = int.Parse(Console.ReadLine());
            Sum = Number1 + Number2;
            Console.WriteLine("The sum of " + Number1 + " and " + Number2 + " is " + Sum + ".");
        }
    }
}

File: java/sum.java
import java.util.Scanner;
class Main 
{
  public static void main(String[] args) 
  {
    System.out.print("To find the sum of two umbers, enter the first number:");
    Scanner scannerObject = new Scanner(System.in);
    int Number1 = scannerObject.nextInt();
    System.out.print("Enter the second number:");
    int Number2 = scannerObject.nextInt();
    int Sum = Number1 + Number2;
    System.out.println("The sum of " + Number1 + " and " + Number2 + " is " + Sum + ".");
  }
}